webcompat/static/js/lib/bugform.js

```diff
@@ -491,8 +499,13 @@ function BugForm() {
       img_url,
       ")"
     ].join("");
+    var delimeter = "[![Screenshot Description](";
     this.descField.val(function(idx, value) {
-      return value + "\n\n" + imageURL;
+      if (value.includes(delimeter)) {
```

Comment:
Just FYI `String.prototype.includes` is a super new JS feature -- I wouldn't recommend using it. You could either use `indexOf` or lodash (since we include that....): https://lodash.com/docs/3.10.1#includes